runtime/oti/j9port.h

```diff
@@ -765,6 +765,9 @@ typedef struct J9CacheInfoQuery {
 #define J9PORT_TTY_OUT OMRPORT_TTY_OUT
 #define J9PORT_TTY_ERR OMRPORT_TTY_ERR
 
+#define J9PRINT_TO_STDOUT "/STDOUT/"
```

Comment:
I don't think this file is the appropriate place for these defines. They are specific to the rasdump code and not the port library. Please put them in an include file specific to the rasdump code.
The defines should also be used in dmpagent.c

Reply:
I've pushed new changes to address your comments. Hopefully I chose the proper file to put the defines in.